apps/challenges/aws_utils.py

```diff
@@ -638,9 +638,8 @@ def scale_resources(challenge, worker_cpu_cores, worker_memory):
     client = get_boto3_client("ecs", aws_keys)
 
     if challenge.worker_cpu_cores == worker_cpu_cores and challenge.worker_memory == worker_memory:
-        message = "Worker cores and memory were not modified."
         return {
-            "Error": message,
+            "IsWorkerConfigModified": True,
```

Comment:
Can you instead send `"Success": True, "message" : "Worker scaled successfully!"`

Reply:
Error messages here aren't propagated to the user; I decided to change the message in views.py corresponding to this case instead.

Reply:
@matthew-so I am saying even if they are not used we should use still follow same schema for each response from this method. We shouldn't introduce new changes to the response message schema. So, basically if we follow the convention of having either a `success` or a `error` key in the response we should stick to that and not add new keys like `IsWorkerConfigModified` Let me know if you have any questions.

Reply:
Got it, thanks!

Reply:
Fixed